### DISCUSSION TITLE
[Questions]About Lora training

### DISCUSSION THREAD

**Discussion by Huli-fox** [ace-step/ACE-Step-1.5#643] [Q&A]
Hi guys, I have deployed this project on an RTX 4090 with a Linux system. I have some questions about Lora training:  
1. Do different DiT models affect the training results?  
2. For the same number of songs (e.g., 10-20 songs), are the recommended epochs the same for different graphics cards?  
3. What is a reasonable loss value after training is complete? I previously tested training on a T4 GPU for 200 epochs and got a loss of 0.5.

**Comment by 1larity** [comment]
You'll be restricted to to using the DiT you trained with so if you train with turbo, you'll only be able to use that LoRA to generate with Turbo.
SFT gives noticeably cleaner generation results than a Turbo  LoRA, and presumably base is better still. 

I've found its better to train with a smaller set of curated songs with a similar style. Don't just throw entire albums with multiple style tracks from the same artist at it. 

Loss value is not the primary gauge of quality, but at the moment, its what you get. Once the loss starts to flatline is a good place to stop, then cherry pick epoch checkpoints every 100 back from when you see the flatline occur. Audition these with a fixed set of generation parameters , fixed seed, same trigger caption and lyrics, all LM features OFF.
the only variance should be the LoRA scale 0.3 to 0.8, every 0.1, and changing the selected cherry picked Epoch, audition one change at a time.

What you are looking for is the sweet spot where you can reliably get the generation style that you want, without overfitting, where the LoRA literally starts ignoring your prompts and will only reproduce the LoRA training data.
